packages/gatsby-theme-nodeschool/src/components/events/upcoming_events.js

```diff
@@ -1,7 +1,10 @@
 import React from "react";
 import { useStaticQuery, graphql } from "gatsby";
+import { useTranslation } from "react-i18next";
+import defaultLanguage from '../../default-language.json';
 
```

Comment:
So instead of this file, can you pull it from the site metadata in the static query below? Is that possible?

Reply:
Yes, It is possible, I will change it.

Reply:
Done.

Reply:
@halkeye , after changed to gatsby-theme-localization on build executed in NodeJS was not working the date format in locale pt-BR (toLocaleDateString), then I fixed it.